features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/.eslintrc.js

```diff
@@ -0,0 +1,69 @@
+module.exports = {
+    parser: 'babel-eslint', // Default parser throws unexpected token error while the syntax is correct
+    parserOptions: {
+        ecmaVersion: 6,
+        ecmaFeatures: {
+            jsx: true,
+            modules: true,
+            experimentalObjectRestSpread: true,
```

Comment:
Are there any rules created on top spread properties ?

Reply:
No but we are using spread property in many places